packages/cozy-harvest-lib/src/components/cards/DeleteAccountCard.jsx

```diff
@@ -26,6 +27,11 @@ export class DeleteAccountCard extends PureComponent {
 
 DeleteAccountCard.propTypes = {
   ...DeleteAccountButton.propTypes,
+  /**
+   * Indicates if the card should be disabled
+   * @type {boolean}
+   */
+  disabled: PropTypes.func,
```

Comment:
It looks like it should be a `PropTypes.bool`

Reply:
Well spotted ! thanks.